cinema/urls.py

```diff
@@ -1 +1,26 @@
-# write urls here
+from rest_framework import routers
+
+
+from cinema.views import (
+    CinemaHallViewSet,
+    GenreViewSet,
+    ActorViewSet,
+    MovieViewSet,
+    MovieSessionViewSet,
+    OrderViewSet,
+    TicketViewSet
+)
+
+router = routers.DefaultRouter()
+router.register("cinema_halls", CinemaHallViewSet)
+router.register("genres", GenreViewSet)
+router.register("actors", ActorViewSet)
+router.register("movies", MovieViewSet)
+router.register("movie_sessions", MovieSessionViewSet)
+router.register("orders", OrderViewSet)
+router.register("tickets", TicketViewSet)
+
+
+urlpatterns = [] + router.urls
```

Comment:
```suggestion
urlpatterns = router.urls
```
You can do like this